skimage/feature/sift.py

```diff
@@ -429,7 +429,7 @@ def _find_localize_evaluate(self, dogspace, img_shape):
 
     def _fit(self, h):
         """Refine the position of the peak by fitting it to a parabola"""
-        return (h[0] - h[2]) / (2 * (h[0] + h[2] - 2 * h[1]))
+        return (h[0] - h[2]) / (2 * ((h[0] - h[1]) + (h[2] - h[1])))
```

Comment:
Arithmetically, it should be exactly the same, but this change prevents "zero" of the case I encountered.

Reply:
Makes sense. Thanks for reporting. Would you mind adding a comment above this line?

Reply:
Sure, I can add a comment.

Reply:
Oh, sorry. I mis-read your comment. Sure, please feel free to add any comments.

Reply:
Thanks, I'll wait for your new commit.